src/app/view/spend_transaction.rs

```diff
@@ -219,32 +330,32 @@ impl SpendTransactionSignView {
         warning: Option<&Error>,
     ) -> Element<'a, Message> {
         let col = Column::new().push(
-            Row::new()
-                .push(
-                    button::transparent(
-                        &mut self.share_button,
-                        button::button_content(None, "Share and update"),
+            Container::new(
+                Row::new()
+                    .push(
+                        button::transparent(
```

Comment:
I don't think it really makes sense to have the "Share and update" button, it would never be used, the "Sign" action is enough IMHO